collectors/cronjob.go

```diff
@@ -150,6 +156,7 @@ func (jc *cronJobCollector) collectCronJob(ch chan<- prometheus.Metric, j v2batc
 	}
 
 	addGauge(descCronJobInfo, 1, j.Spec.Schedule, string(j.Spec.ConcurrencyPolicy))
+	addGauge(descCronJobCreated, float64(j.CreationTimestamp.Unix()))
```

Comment:
Argh, sorry this was incorrect from my side. I read from the code comment that this `nil`'able, but in fact it's not a pointer actually, so what you did at first, was actually correct (* hides in shame *).